### ISSUE TITLE
Problems with Ragnarok 

### ISSUE THREAD

**Issue by Onatel** [thommcgrath/Beacon#130]
Hi,

I use these community config: 

https://beaconapp.cc/browse/view.php?document_id=9c70c644-5fe3-4118-8c7d-8a8e310659c6&map_filter=8

And I have the problem that the beacons don´t spawn correctly. The sourced earth beacons in the source earth areas of ragnarok [desert for example] spawn correctly, same with the beaver spots. They all have the primal fear items in it.
If I spawn as admin for example a *white beacon* in Vikingbay with these spawn command "admincheat summon  SupplyCrate_Level03_C", it also has the correct items in it. But if the game spawn these beacons, the items are vanilla. 
I have tested and checked a lot of things. Wrote the config in the first place, nothing changed. Doublechecked all spawn names, nothing goes wrong. 
But the beacons in the island area of ragnarok don´t spawn correctly. 
I´m desperate.

I don´t use nitrado. I have my own linux root server with ARK installed as dedicated server.
No FTP is allowed, so I can´t use the beacon upload function.

I hope anybody can help.

MfG Onatel

**Comment by thommcgrath** [comment]
If the crate spawns with correct items with the cheat code, then it's working. So I would use the `debugstructures` cheat code while looking at a drop that has the wrong contents to figure out what crate code it is using.

**Comment by Onatel** [comment]
Thx for the quick reply. I will test and report what I get :)

**Comment by Onatel** [comment]
Thx for the quick reply. I will test and report what I get :)

**Comment by Onatel** [comment]
Yes, that was the right hint 💃 .
I found my mistake.
In the first load order, I use the mod Ark Eternal, these mod also override some beacons.
So all vanilla beacons have a AE between the level and the _C, 
for example: "SupplyCrate_Level15_**AE**_C" instead of "SupplyCrate_Level15_C".
So I changed all the vanilla-beacons to the AE-beacons and everthing is fine. 
All beacons spawn now correctly.

Thank you very much for your help 👍 
Have a nice day.
